libs/game/game.ts

```diff
@@ -111,8 +111,6 @@ namespace game {
      * @param win whether the animation should run on a win (true)
      * @param effect
      */
-    //% group="Gameplay"
-    //% blockId=setGameOverEffect block="set game over effect for win %win=toggleYesNo to %effect"
     export function setGameOverEffect(win: boolean, effect: effects.BackgroundEffect) {
```

Comment:
We automatically do gameover when life == 0, so we need this block to configure the effect.
https://github.com/Microsoft/pxt-common-packages/pull/634/files#diff-7c3ef1deff0b4e062963ed4e50585d99L116